CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 **Gem enhancements:**
 
+- Reduced production gem size by 9 times [[#345](https://github.com/panorama-ed/memo_wise/pull/345)]
```

Comment:
To be honest I always find statements like this confusing (would `reduced size by 1x` mean no reduction or a 100% reduction?). Would you be open to changing this to absolutes, e.g.:

```
Reduced unzipped gem size from 173KB to 19KB
```

or something similar?

Reply:
Much better, thanks